vendor/bat-native-ads/src/bat/ads/internal/account/account.cc

```diff
@@ -327,8 +327,13 @@ void Account::OnDidRefillUnblindedTokens() {
 void Account::OnCaptchaRequiredToRefillUnblindedTokens(
     const std::string& captcha_id) {
   const WalletInfo& wallet = GetWallet();
+#if BUILDFLAG(IS_ANDROID)
+  AdsClientHelper::GetInstance()->ShowScheduledCaptchaAndroid(wallet.id,
```

Comment:
I think it's cleaner to just rename the function `ShowScheduledCaptcha` and pass a Boolean `show_notification` flag which would be `false` on Android and `true` elsewhere. Otherwise, I think we'll unnecessarily pollute the interface with Android-specific function variations.